specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2021-09-01-preview/AlertRules.json

```diff
@@ -748,6 +758,43 @@
       },
       "type": "object"
     },
+    "QueryBasedAlertRuleTemplateProperties": {
```

Comment:
what about the template version?

Reply:
when you'll complete your PR I'll be able to add it
currently id does not exist

Reply:
yes